Cargo.toml

```diff
@@ -12,7 +12,6 @@ members = [
     "tests/import_contract",
     "tests/invoke_contract",
     "tests/udt",
-    "tests/contract_data",
```

Comment:
Why is the contract data example removed?

Reply:
I re-added in b1fcf44.

Reply:
Ah yeah this was a mistake. Thanks for fixing it.